src/rabbit_memory_monitor.erl

```diff
@@ -86,7 +86,9 @@ report_ram_duration(Pid, QueueDuration) ->
 stop() ->
     gen_server2:cast(?SERVER, stop).
 
-conserve_resources(Pid, disk, Conserve) ->
+%% Paging should be enabled/disabled only in response to disk resource alarms
```

Comment:
> as discussed on slack with @dumbbell what would happen when node A that doesn't have an alarm, receives a publish that has to be routed or replicated to node B (who has a disk alarm for instance). What to do here?

Are we all discussing the same thing? ) 

There is no significant change in alarm behaviour, here is the picture with explanation what the guard expression on following line changes:

![alarm-seq](https://cloud.githubusercontent.com/assets/185443/10795251/fcf544f6-7da9-11e5-94c2-fba3fdfddb1d.png)